final_task/rss_reader/functions/print_func.py

```diff
@@ -0,0 +1,100 @@
+"""Contain formatting output functions"""
+
+import json
+import classes.exceptions as exc
+
+
+def check_limit_argument(command_line_args, news_collection, logger):
```

Comment:
It is better to give this function another name, which better describes what it does.

Reply:
Fixed